Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -497,9 +497,9 @@
 		G_8620238527590 /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				FR_825232110500 /* App_iOS.app */,
 				FR_783122899910 /* App_iOS_Tests.xctest */,
 				FR_123503999387 /* App_iOS_UITests.xctest */,
+				FR_825232110500 /* App_iOS.app */,
```

Comment:
Assuming this was out of date?

Reply:
Yes.

Reply:
Yes, it seems since #316, tests haven't been failing properly on CI.
That means the required fixture changes weren't reported as a failure as part of #341.

The test framework has been fixed, and this file has been update in master now 👍